manifests/r/Ravencoin/RavenCore/4.6.1/Ravencoin.RavenCore.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-608.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Ravencoin.RavenCore
+PackageVersion: 4.6.1
+PackageLocale: en-US
+Publisher: RavenCore
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: Raven Core Client/Wallet
```

```suggestion
PackageName: Raven Core
```
Name should match with Control Panel.